core/chains/evm/logpoller/log_poller_internal_test.go

```diff
@@ -393,8 +396,8 @@ func TestLogPoller_Replay(t *testing.T) {
 		}()
 		select {
 		case <-timeout:
-			assert.Fail(t, "lp.run() got stuck--failed to respond to second replay event within 2s")
-		case <-utils.WaitGroupChan(&wg):
+			assert.Fail(t, "lp.run() got stuck--failed to respond to second replay event within %s", testutils.WaitTimeout(t))
```

Comment:
Calling `testutils.WaitTimeout` again at this point will return a different value than was actually used, because it is computed based on remaining time until the deadline.

Reply:
Oh, good point.  Just realized, I also need to add a `fmt.Sprintf()` otherwise this %s isn't going to work.